more_itertools/more.py

```diff
@@ -431,6 +431,9 @@ def __getitem__(self, index):
 
         return self._cache[index]
 
+    def __class_getitem__(cls, key):
+        return cls
+
```

Comment:
This should be: `__class_getitem__ = classmethod(GenericAlias)`

Also suggest adding a test to make sure it works: `self.assertEqual(peekable[str].__args__,
(str,))`